.gitignore

```diff
@@ -1,7 +1,11 @@
+# godot
+
 /.import
+*.import
```

Comment:
*.import files shouldn't be ignored - they store metadata that is necessary for assets to be imported/used correctly.

Reply:
Oh okay, thanks!